src/redux/fake-api-data/FakeApiDataReducer.js

```diff
@@ -0,0 +1,55 @@
+import {
+  SET_DATE_RANGE,
+  SetCount,
+  SetData,
+  SetDate,
+  SetLoading,
+} from "./fakeApiDataTypes";
+
+const initialState = {
+  data: [],
+  loading: true,
+  count: null,
+  startDate: null,
+  endDate: null,
+};
+
+const FakeApiReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SetData: {
```

Comment:
@9xVibee can you change these casing instead of Pascal case can you use All caps like SET_DATA